[fix bug 1136934] Firefox 38.0.5 whatsnew page

alexgibson opened the pull request:
Three different templates depending on the locale:

* Video plus Pocket - `/en-US/firefox/38.0.5/whatsnew/`
* Video only - `/fr/firefox/38.0.5/whatsnew/`
* Pocket only - `/ja/firefox/38.0.5/whatsnew/`

flodolo commented on bedrock/firefox/templates/firefox/whatsnew_38/whatsnew-base.html:
```diff
@@ -0,0 +1,59 @@
+
+{% extends "firefox/base-resp.html" %}
+
+{% add_lang_files "firefox/38_1_whatsnew" %}
```
If we need it, I'd use the same ```firefox/whatsnew_38```

alexgibson replied:
Thanks, meant to update this one when I renamed things

flodolo commented on bedrock/firefox/templates/firefox/whatsnew_38/whatsnew-a.html:
```diff
@@ -0,0 +1,46 @@
+  <section id="video-content">
+    <h1 class="large">{{_('From our hands to yours')}}</h1>
+    <p class="tagline">{{_('Firefox is built from the heart by thousands who believe in personal freedom on the Web.')}}</p>
+    <a id="video-thumbnail" href="{{ fx_38_video_url}}.mp4">{{_('Watch Video')}}</a>
```
Can we use ```Watch the video```, which is the string we used so far and already available in main.lang?

alexgibson replied:
Sure +1

alexgibson commented:
Still waiting on the final localized videos, plus poster image - but this should now be good for code review.

alexgibson commented:
Still missing de, es-MX, and pt-BR videos.

ckprice commented:
All videos and poster image are [posted](https://bugzilla.mozilla.org/show_bug.cgi?id=1140042).

ckprice commented:
@jpetto r?

jpetto commented on bedrock/firefox/templates/firefox/whatsnew_38/whatsnew-pocket-video.html:
```diff
@@ -0,0 +1,55 @@
+
+{% block content %}
+<main role="main">
+    <section id="pocket">
```
Nit - extra indentation.

jpetto commented on media/js/firefox/whatsnew_38/whatsnew-video.js:
```diff
@@ -0,0 +1,43 @@
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+$(function(){
```
Do these need to be in a `document.ready` handler?

alexgibson replied:
Maybe not, but I think there's very little gain here for this page to make them immediately invoked?

jpetto replied:
No big gain, but one less event handler and keeping consistent with other JS files.

alexgibson replied:
Hmmm.... okay you persuaded me :+1:

jpetto commented:
Since we're using the same video poster image here and in PR #2932, we should probably merge the images to master first, then rebase the PRs. I can open a PR for the images.

alexgibson commented:
@jpetto - rebased and using common poster images for video

jpetto commented:
Looking good. Squashing please?

alexgibson commented:
Squarshed!

alexgibson commented:
@jpetto - rebased and resolved conflicts. I renamed the function in `/firefox/views.py` to `show_38_0_5_firstrun_or_whatsnew` and updated accordingly.

jpetto commented:
r+! :haircut:
